tests/test_interactions.py

```diff
@@ -181,7 +181,12 @@ def test_score_t_slicing(self):
         )
         scores = self.instance.score_t(h=h, r=r, all_entities=t, slice_size=self.num_entities // 2 + 1)
         scores_no_slice = self.instance.score_t(h=h, r=r, all_entities=t, slice_size=None)
-        assert torch.allclose(scores, scores_no_slice), f'Differences: {scores - scores_no_slice}'
+        self._check_close_scores(scores=scores, scores_no_slice=scores_no_slice)
+
+    def _check_close_scores(self, scores, scores_no_slice):
+        self.assertFalse(torch.isnan(scores).any(), msg=f'Normal scores had nan:\n\t{scores}')
```

Comment:
@cthoyt there is also `torch.isfinite` additionally checking for +/- inf

Reply:
Nice, should we add that in addition to the nan check? Or do you mean instead?

Reply:
I think instead is fine